README.ksonnet.md

```diff
@@ -0,0 +1,183 @@
+# Using Ksonnet 
+
+Kubeflow is currently evaluating different strategies for allowing deployments to be customized ([google/kubeflow/issues/23](https://github.com/google/kubeflow/issues/23)).
+
+This directory contains a prototype for using ksonnet to help us evaluate ksonnet.
+
+If you are unfamiliar with ksonnet you may want to start by reading the [tutorial](https://ksonnet.io/docs/tutorial)
+
+## Build ks
+
+ksonnet doesn't support adding non default registries yet ((ksonnet/ksonnet/issues/38)[https://github.com/ksonnet/ksonnet/issues/38]) so we need to modify and [build from source](https://ksonnet.io/docs/build-install).
+
+```
+# Clone the ksonnet repo into your GOPATH
+go get github.com/ksonnet/ksonnet
+```
+
+Checkout PR 228 which adds support for registry add
```

Comment:
This has been merged: https://github.com/ksonnet/ksonnet/commit/29e3cd455beea7557f4f81430e9c6ef635444335